common-src/amcrypt-ossl-asym.sh

```diff
@@ -164,7 +164,7 @@ decrypt() {
 	[ $? -eq 0 ] || return 1
 
 	# use the cipher key to decrypt data
-	pad | "${OPENSSL}" enc "-${CIPHER}" -nopad -d -pass "file:${WORKDIR}/pass" -nosalt
+	pad | "${OPENSSL}" enc -pbkdf2 "-${CIPHER}" -nopad -d -pass "file:${WORKDIR}/pass" -nosalt
```

Comment:
Do that break compatibility?
Did it decrypt backup done without the -pbkfd2 option?

Reply:
valid question, didn't yet test that.
Maybe it would be enough to only use that option in the "encrypt" section (to remove the warning in the report emails)?

Reply:
> Do that break compatibility?
> Did it decrypt backup done without the -pbkfd2 option?

Yes, you are right: with the option in the script it can't restore backups encrypted with the earlier version of the script. This is breaking things.

I assume there might be some way to detect the format of the data to be decrypted and add some if-clause ?